Resources/views/MediaAdmin/multi_upload.html.twig

```diff
@@ -0,0 +1,137 @@
+{% extends 'SonataAdminBundle:CRUD:base_edit.html.twig' %}
+
+{% block stylesheets %}
+    {{ parent() }}
+    <link rel="stylesheet" href="{{ asset('bundles/sonatamedia/multi_upload/css/jquery.fileupload.css') }}">
+    <link rel="stylesheet" href="{{ asset('bundles/sonatamedia/multi_upload/css/jquery.fileupload-ui.css') }}">
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+    <script src="{{ asset('bundles/sonatamedia/multi_upload/js/load-image.all.min.js') }}"></script>

```

Comment:
Is it possible to add these lines to the ```sonata_admin.assets.javascripts``` configuration in some way? 
This would allow better overview about used/loaded javascripts (and stylesheets above), without the need to override templates in some cases.